dotnet/src/Connectors/Connectors.Onnx/Connectors.Onnx.csproj

```
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.Onnx</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFramework>net6.0</TargetFramework> <!-- TODO: Support netstandard2.0 once replacement for FastBertTokenizer available -->
+    <VersionSuffix>preview</VersionSuffix>
```

Is there a special reason to promote this connector to be a preview right away?
```suggestion
    <VersionSuffix>alpha</VersionSuffix>
```